docs/serviceability.md

```diff
@@ -0,0 +1,76 @@
+# Serviceability
+
+The serviceability document aims to help local application-service developers and SRE engineers to access and service the application-service component. This document will help you understand how to access and understand the application-service logs, how to debug an application-service problem and provides a quick summary on the various questions that you might have regarding application-service. 
+
+## Accessing the Logs
+
+### Deployed Locally
+View the application-service controller logs by running the executable manager. Example,
+
+```
+make install
+make build
+./bin/manager
+```
+
+### Deployed on a Local Cluster
+View the application-service controller logs by tailing the manager container log of the controller manager pod. The pod resides in the application-service namespace. Example,
+
+```
+oc logs -f application-service-application-service-controller-manager -c manager -n application-service
+```
+
+### Deployed on a Managed Cluster
```

Comment:
Perhaps just leave this section out since it’s RHTAP specific

Reply:
I will probably keep it in for now, because Kim suggested we put it here in her PR review. I think we can iterate over it and move it in the future if we dont want it.